src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -159,15 +154,15 @@
           expect(response).to include(
             'name'              => 'openSUSE:Factory:Staging:A',
             'description'       => description,
-            'obsolete_requests' => [JSON.parse(bs_request.to_json)],
+            'obsolete_requests' => [JSON.parse(declined_bs_request.to_json)],
             'overall_state'     => 'unacceptable'
           )
         end
       end
     end
 
     context 'with a non-existent factory_staging_project' do
-      subject { get :show, params: { project: factory, project_name: 'B' } }
+      subject { get :show, params: { project: factory, project_name: 'C' } }
```

Comment:
what was the problem with `B`? :unamused:

Reply:
Ring `B` was used by some of the other tests. Since this got moved up I had to change this to C.

Reply:
:unamused: maybe using a more original name which avoid future conflicts? :thinking:

Reply:
Let's use `Faker`. Something like `Faker::String.random(10)`

Reply:
It's aligned to what is currently used  for the staging project rings. That said I am also fine with changing it, but not in the PR as this is unrelated:-)